docs/pages/machine-id/guides/hostcertificate.mdx

```diff
@@ -0,0 +1,373 @@
+  ```
+  </TabItem>
+  <TabItem label="IAM Method">
+  First, create an IAM method token that specifies the AWS account from which
+  the bot can join. Create the below file as `iam-token.yaml` then run `tctl
+  create -f iam-token.yaml`.
+
+  ```
+  kind: token
+  version: v2
+  metadata:
+    # The token name is not a secret because instances must prove that they are
+    # running in your AWS account to use this token.
+    name: iam-token
+    # Set a long expiry time for how long you want to support IAM method for
+    # joining. It is safe to set this value to a very long time.
+    expires: "3000-01-01T00:00:00Z"
+  spec:
+    # Only allow bots to join using this token.
+    roles: [Bot]
+
+    # Set the join method to be IAM.
+    join_method: iam
+
+    # Define the name of the bot that will be allowed to use this token.
+    bot_name: robot
+
+    allow:
+    # Restrict the AWS account and (optionally) ARN that can use this token.
+    # This information can be obtained from running the
+    # "aws sts get-caller-identity" command from the CLI.
+    - aws_account: "111111111111"
+      aws_arn: "arn:aws:sts::111111111111:assumed-role/teleport-bot-role/i-*"
+  ```
+
+  Next, create the bot user.
+
+  ```
+  $ tctl bots add robot --token=iam-token --roles=hostcert-bot
+  ```
+  </TabItem>
+</Tabs>
+
+Machine ID may now be configured using the `tbot configure` command on the bot node.
+
+First, create a basic configuration file using the following parameters:
+
+<Tabs>
+  <TabItem label="Token-based Joining">
+
+  <ScopedBlock scope={["oss", "enterprise"]}>
+
+  ```code
+  $ tbot configure \
+     --data-dir=/var/lib/teleport/bot \
+     --token=(=presets.tokens.first=) \
+     --join-method=token \
+     --certificate-ttl=1h0m0s \
+     --ca-pin=(=presets.ca_pin=) \
+     --auth-server=auth.example.com:3025
+  ```
+
+  </ScopedBlock>
+  <ScopedBlock scope={["cloud"]}>
+
+  ```code
+  $ tbot configure \
+     --data-dir=/var/lib/teleport/bot \
+     --token=(=presets.tokens.first=) \
+     --join-method=token \
+     --certificate-ttl=1h0m0s \
+     --ca-pin=(=presets.ca_pin=) \
+     --auth-server=example.teleport.sh:443
+  ```
+
+  </ScopedBlock>
+
+  </TabItem>
+  <TabItem label="IAM Method">
+
+  <ScopedBlock scope={["oss", "enterprise"]}>
+
+  ```code
+  $ tbot configure \
+     --data-dir=/var/lib/teleport/bot \
+     --token=iam-token \
+     --join-method=iam \
+     --certificate-ttl=1h0m0s \
+     --ca-pin=(=presets.ca_pin=) \
+     --auth-server=auth.example.com:3025
+  ```
+
+  </ScopedBlock>
+  <ScopedBlock scope={["cloud"]}>
+
+  ```code
+  $ tbot configure \
+     --data-dir=/var/lib/teleport/bot \
+     --token=iam-token \
+     --join-method=iam \
+     --certificate-ttl=1h0m0s \
+     --ca-pin=(=presets.ca_pin=) \
+     --auth-server=example.teleport.sh:443
+  ```
+
+  </ScopedBlock>
+
+  </TabItem>
+</Tabs>
+
+Replace the following fields with values from your own cluster.
+
+<ScopedBlock scope={["cloud"]}>
```

Comment:
I would recommend using `Tabs` instead of a `ScopedBlock` since it's not clear that a `ScopedBlock` has variations for different scopes, and users often (if not usually) don't realize that they can switch scope.